config/webpack/configuration.js

```diff
@@ -12,7 +12,7 @@ const devServer = safeLoad(readFileSync(join(configPath, 'development.server.yml
 
 // Compute public path based on environment and CDN_HOST in production
 const ifHasCDN = env.CDN_HOST !== undefined && env.NODE_ENV === 'production';
-const devServerUrl = `http://${devServer.host}:${devServer.port}/${paths.entry}/`;
+const devServerUrl = `http://${env.LOCAL_DOMAIN || devServer.host}:${devServer.port}/${paths.entry}/`;
```

Comment:
Could we create `development.server.yml` for Nanobox as we do for `.env`, or override `NODE_ENV` at runtime?
I do not think it is good to have an extra environment variable for a specific environment while we have a configuration file for the same purpose.